docs/data/material/components/autocomplete/LimitTags.tsx

```diff
@@ -14,6 +14,7 @@ export default function LimitTags() {
       renderInput={(params) => (
         <TextField {...params} label="limitTags" placeholder="Favorites" />
       )}
+      sx={{ minWidth: '500px' }}
```

Comment:
I don't think that it's working because:

1. It overflows

<img width="552" alt="Screenshot 2022-02-07 at 01 30 51" src="https://user-images.githubusercontent.com/3165635/152708455-3944a654-03e2-43d4-8bfa-57afc469235c.png">

2. It moves

https://user-images.githubusercontent.com/3165635/152708517-2361e127-bb64-4e50-89d4-fc3b1d3b094b.mp4

```suggestion
      sx={{ width: '500px' }}
```

Reply:
@danilo-leal It is unclear to me what you want to fix in the demo. Can you explain a bit more?

Reply:
@siriwatknp If you go to the _Limit tags_ demo and remove all chips/tags, you'll see that the label text overflows the Autocomplete container. The added width is just to fix this behavior when there are no chips.